src/wp-content/themes/twentyeleven/showcase.php

```diff
@@ -208,7 +208,11 @@
 					if ( $recent->have_posts() ) :
 						$recent->the_post();
 
-						// Set $more to 0 in order to only get the first part of the post.
+						/*
```

Comment:
single-line comment style changed to multi-line (should it be docblock or edited another way?)